doc/src/vtr/server_mode/index.rst

```diff
@@ -202,7 +202,17 @@ Draw selected critical path elements example
 
   .. note:: If status is not 1, the field ***DATA*** contains error string.
 
+.. _ipa_client_for_testing_vpr_server_mode_functionality:
```

Comment:
I'd extend this a bit to: "The interactive path analysis (IPA) client is useful for viewing timing paths, and to test VPR's server mode functionality" (with the link embedded somewhere).

Reply:
resolved, 041e177e58650dd0196f00d7247ed0d6c9e9591f